pkgs/applications/video/gnome-subtitles/default.nix

```
@@ -0,0 +1,83 @@
+{ stdenv
+, fetchFromGitLab
+, autoreconfHook
+, pkg-config
+, intltool
+, itstool
+, mono
+, yelp-tools
+, wrapGAppsHook
+, enchant1
+, glib
+, gnome3
+, gtk-sharp-3_0
+, gst_all_1
+, libxml2
+}:
+let
+  version = "1.6";
+in
+stdenv.mkDerivation {
+  pname = "gnome-subtitles";
+  inherit version;
```

personal preference, don't feel strongly about it
```suggestion
stdenv.mkDerivation rec {
  pname = "gnome-subtitles";
  version = "1.6";
```

---

Thanks for suggestion. Applied